src/lib/schemaParser.ts

```diff
@@ -88,7 +88,10 @@ const fillDynamicKeyPart = (
 
     // if first 20 bytes of the hash of second word in schema match,
     // display the map part as plain word
-    if (keccak256(keyNameParts[1]).substring(0, 42) === `0x${secondWordHex}`) {
+    if (
+      keccak256(toBytes(keyNameParts[1])).substring(0, 42) ===
```

Comment:
Couldn't we use a hex slicing function from viem? It would be cleaner here over `substring`

Reply:
Yes we could for sure. Checking into it.